SwiftCompilerSources/Sources/SIL/Argument.swift

```diff
@@ -34,7 +34,12 @@ public class Argument : Value, Hashable {
 
   public var isLexical: Bool { false }
 
-  public var varDecl: VarDecl? { bridged.getVarDecl().getAs(VarDecl.self) }
+  public var varDecl: VarDecl? {
```

Comment:
I don't think it's good to combine `getDecl` and the lookup of `debug_value` users into a single API.
I think it's better to directly bridge `getDecl` (returning a `Decl`) and add `varDecl` on top of this. And `varDecl` belongs more to OptUtils than defining it in the "raw" SIL, because it looks at a bigger SIL structure (including the users) than the Argument in isolation.